libbeat/setup/kibana/client.go

```diff
@@ -65,6 +65,11 @@ func NewKibanaClient(cfg *common.Config) (*Client, error) {
 		return nil, err
 	}
 
+	return NewClientWithConfig(&config)
+}
+
+// NewClientWithConfig creates and returns a kibana client using the given config
+func NewClientWithConfig(config *Config) (*Client, error) {
```

Comment:
Not relate to this PR but it seems Kibana becomes more then just for setup. Perhaps we should move this into `libbeat/kibana/client` in the future.

Reply:
I was planning to extract the kibana code to a different PR anyway, will move it :+1: